whipper/command/cd.py

```diff
@@ -296,13 +296,11 @@ def add_arguments(self):
         self.parser.add_argument('--track-template',
                                  action="store", dest="track_template",
                                  default=DEFAULT_TRACK_TEMPLATE,
-                                 help="template for track file naming "
-                                 "(default default)")
+                                 help="template for track file naming")
```

Comment:
Maybe the default value should be kept but in a correct way (not default default).

Reply:
It's already displayed.

```  --track-template TRACK_TEMPLATE
                        template for track file naming (default default)
                        (default: %r/%A - %d/%t. %a - %n)
  --disc-template DISC_TEMPLATE
                        template for disc file naming (default default)
                        (default: %r/%A - %d/%A - %d)
```

Reply:
Great, I suspected I was missing something obvious. Sorry for the useless remark!